### ISSUE TITLE
💡 [FEAT] Command to point towards a specific .db file

### ISSUE THREAD

**Issue by jttasso** [dooit-org/dooit#224]
I just noticed that dooit didn't have the option of having various separate projects I can point to which seems odd since something like that could very well be vital for organization.

I don't actually know how others point their dooit towards different db-files, or if they simply do everything from a singular database.

Having an option like pointing towards a file would be fantastic as far as organization and compartmentalization is concerned.

Something like:
dooit -p /Path/to/Database
would be absolutely huge.

Dunno how feasible a database switcher inside an opened dooit instance would be but I imagine at the very least the path pointing to be reasonable.

**Comment by kraanzu** [comment]
Yep. Some other person also suggested this, I was thinking something like `profiles`

you can create a profile with a name and db path
and launch something like:

```
dooit -p jttasso 
```

what do you think?

**Comment by jttasso** [comment]
Couple thoughts:
Since from what you wrote it seems that the profile only has 1 db path, right? But since the db file most likely only concerns itself with a singular project or field due to only 1 db path, why come up with the profiles part at all?
Unless of course you have something more intricate planned that I am not aware of.

IF the profile only points towards 1 db path OR it does point towards multiple paths without any option to switch between them seamlessly, I feel it more sensible to simply have a command to point towards a db-file to be opened as I've suggested earlier.

Reason I think so is: if I wanted to quickly open various databases with what you have suggested, 
1. I would need to first create a profile (via a config file or something I assume) 
2. type in that command with the profile
3. (for further customization) create an alias in my bashrc or zshrc file.

With a path command like `dooit -p /path/to/database` I would only need to create an alias.

For simplicity and customizability sake I think the path command seems to be the way to go.
Though of course my argument would quickly fall apart depending on how exactly you plan to integrate your profiles.
How indeed do you want to integrate these profiles?

Though, either way, simply having a way to point towards varying databases in itself would be pretty baller.

**Comment by kraanzu** [comment]
Yep same thought, I'll just do both. Since profile is just a nice sugar coat for the underlying command

So you'd be able to do:

```
dooit --db ~/super_secret_project.db
```

**Comment by jttasso** [comment]
Oooh yeah thats it. Now that's what I'm talking about. Simple yet elegant and functional. What else could anybody be asking for.

**Comment by kraanzu** [comment]
Sorry got delayed, My life was turned upside down lol
Check out --help menu, this is supported now
